mailer.js

```diff
@@ -11,6 +12,24 @@ var DEFAULT_LOCALE = 'en'
 var DEFAULT_TIMEZONE = 'Etc/UTC'
 
 module.exports = function (log) {
+  // Email template to UTM campaign map
+  var templateNameToCampaignMap = {
+    'passwordResetRequiredEmail': 'password-reset-required',
+    'passwordChangedEmail': 'password-changed-success',
+    'passwordResetEmail': 'password-reset',
+    'postVerifyEmail': 'account-verified',
+    'newDeviceLoginEmail': 'new-device-signin',
+    'recoveryEmail': 'forgot-password',
+    'resetEmail': 'forgot-password',
+    'suspiciousLocationEmail': 'suspicious-location',
+    'verifyEmail': 'welcome',
+    'verifyLoginEmail': 'new-signin',
+    'verificationReminderFirstEmail': 'hello-again',
+    'verificationReminderSecondEmail': 'still-there',
+    'verificationReminderEmail': 'hello-again'
+  }
+  var utmPrefix = 'fxa-'
```

Comment:
Similar to DEFAULT_TIMEZONE above, this is a named constant that renames the same, could you rename it to `UTM_PREFIX` for clarity? I had to search to find where this was declared when I saw it used below.